.github/workflows/firebase-hosting-merge.yml

```diff
@@ -0,0 +1,28 @@
+# This file was auto-generated by the Firebase CLI
+# https://github.com/firebase/firebase-tools
+
+name: Deploy to Firebase Hosting on merge
+'on':
+  push:
+    branches:
+<<<<<<< HEAD
+      - API
+=======
+      - master
+>>>>>>> a2612daa31d6922012b3c769eb516c580cf83b43
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm ci && npm run build
+      - uses: FirebaseExtended/action-hosting-deploy@v0
+        with:
+          repoToken: '${{ secrets.GITHUB_TOKEN }}'
+          firebaseServiceAccount: '${{ secrets.FIREBASE_SERVICE_ACCOUNT_COP4331_LARGE_PROJECT_27 }}'
+          channelId: live
+          projectId: cop4331-large-project-27
+<<<<<<< HEAD
+
+=======
+>>>>>>> a2612daa31d6922012b3c769eb516c580cf83b43
```

Comment:
Lol this is def what its not that's supposed to look like, just delete the workflow stuff
since we deploy manually